File: backend/src/main/java/com/snackoverflow/toolgether/domain/reservation/dto/ReservationRequest.java
package com.snackoverflow.toolgether.domain.reservation.dto;

import java.time.LocalDateTime;

public record ReservationRequest(
	Long postId,
	Long renterId,
	Long ownerId,
	LocalDateTime startTime,
	LocalDateTime endTime,
	Double deposit,
	Double rentalFee
) {}

File: backend/src/main/java/com/snackoverflow/toolgether/domain/reservation/dto/ReservationRequest.kt
package com.snackoverflow.toolgether.domain.reservation.dto

import java.time.LocalDateTime

@JvmRecord
data class ReservationRequest(
    val postId: Long,
    val renterId: Long,
    val ownerId: Long,
    val startTime: LocalDateTime,
    val endTime: LocalDateTime,
    val deposit: Double,
    val rentalFee: Double
)